scripts/travis/test.sh

```diff
@@ -14,6 +14,8 @@ curl -sL -o ~/gimme https://raw.githubusercontent.com/travis-ci/gimme/master/gim
 chmod +x ~/gimme
 eval "$(~/gimme "${GOLANG_VERSION}")"
 
+"${SCRIPTPATH}/../buildtools/install_buildtools.sh" "gotest.tools/gotestsum"
```

Comment:
could you allow this step to fail ? It would default to the `go test` in that case, but would not fail the build if this is no longer available.

Reply:
I could, but is that really the desired behavior? If our build tools aren't working I think we should fix them, if I allow this to fail we may not know that there's a problem.

Reply:
I want to avoid ( when possible ) build failure due to "internet" random failures. If this is not a crucial component, we shouldn't make it into one. The rest of the pipeline would work correctly in either way.

Reply:
In general, I disagree with this approach for the development workflow. It's how we've done things for a while and has led to things being broken for extended periods because things still mostly work and nobody noticed. We should design the system so that flaky components are annoying, that way they get noticed and fixed.